src/recipes/unsupportedBrowser.js

```diff
@@ -0,0 +1,26 @@
+/* 
+    RECIPE: unsupported browser
+    -------------------------------------------------------------
+    Author: Morgan Graham, Lia Hiscock
+    Description: Looking for phrases that tell users that Edge is not supported, or to switch browers. 
+*/
+
+void function() {
+    window.CSSUsage.StyleWalker.recipesToRun.push( function unsupportedBrowser( element, results) {        
+        //tests for phrases
+        var switchPhraseString = new RegExp("(Switch to|Get|Download|Install)(\\w|\\s)+(Google|Chrome|Safari|firefox|Opera|Internet Explorer)","i");
```

Comment:
You either need to be checking for a variety of character casings or convert them yourself using a DOM API. This is what I'd recommend since CSS can actually change the visual representation of the content to be Chrome while in the DOM it will chrome.